llama_hub/web/main_content_extractor/README.md

```diff
@@ -0,0 +1,67 @@
+# MainContentExtractor Website Loader
+
+This loader is a web scraper that fetches the text from static websites using the `MainContentExtractor` Python package.
+
+For information on how to extract main content, README in the following github repository
+
+[HawkClaws/main_content_extractor](https://github.com/HawkClaws/main_content_extractor)
+
+## Usage
+
+To use this loader, you need to pass in an array of URLs.
+
+```python
+from llama_index import download_loader
+
+MainContentExtractorReader = download_loader("MainContentExtractorReader")
+
+loader = MainContentExtractorReader()
+documents = loader.load_data(urls=['https://google.com'])
+```
+
+## Examples
+
+### LlamaIndex
+
+```python
+from llama_index import GPTVectorStoreIndex, download_loader
```

Comment:
GPTVectorStoreIndex is deprecated, can you change to VectorStoreIndex?

Reply:
Thanks for pointing that out.
GPTVectorStoreIndex has been changed to VectorStoreIndex